airbyte-integrations/connectors/source-tiktok-marketing/acceptance-test-config.yml

```diff
@@ -22,30 +25,52 @@ tests:
       status: "failed"
     - config_path: "integration_tests/invalid_config_oauth.json"
       status: "failed"
+
   discovery:
-    - config_path: "secrets/config.json"
     - config_path: "secrets/prod_config.json"
+    - config_path: "secrets/prod_config_day.json"
+    - config_path: "secrets/config.json"
     - config_path: "secrets/new_config_prod.json"
     - config_path: "secrets/config_oauth.json"
+
   basic_read:
-    - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/configured_catalog.json"
-      empty_streams: [ "ads" , "ads_reports", "ad_groups_reports", "campaigns_reports" ]
+    # DAY granularity:
+    - config_path: "secrets/prod_config_day.json"
+      configured_catalog_path: "integration_tests/streams_all.json"
+      timeout_seconds: 1200
+      validate_schema: false  # validation fails for audience report streams
```

Comment:
Are we aware of what exactly is wrong with the validation of `Audience Report` streams?